docs/source/intro.md

```diff
@@ -0,0 +1,59 @@
+```{currentmodule} batconf
+```
+
+```{toctree}
+:hidden: true
+:maxdepth: 2
+```
+
+# Introduction
+
+Designed to provide 12-factor compliant configuration management
+for python microservices, applications, and automation tools.
+
+Provides builtin support for hierarchical configuration via:
+* CLI args
+* Environment Variables
+* Config File (yaml)
+* Config classes with default values
+
+Users can create their own config sources
+by creating classes that satisfy `batconf.source.SourceInterfaceProto`
+(or subclass `batconf.source.SourceInterface`)
+
+The config lookup order is determined by the `SourceList` instance,
+which can be adjusted to suit your needs.
+
+## Motivation
+
+This module was created to serve the configuration needs of python projects,
+esp; micro-services, command line applications, and service clients.
```

Comment:
...configuration needs of python projects, especially: micro-services, command...

Reply:
fixed